### PR TITLE
add swapx spec

### PR DESCRIPTION
**Pull request by DreamWuGit**

### CODE REVIEW

File: src/zkevm_specs/opcode/stack.py [hunk 1]
```diff
@@ -48,4 +58,19 @@ def check_dupx():
     evm_stack.push(word2)
     assert(word2 == evm_stack.peek())
     evm_stack.dupx(2)
-    assert(word1 == evm_stack.peek())
\ No newline at end of file
+    assert(word1 == evm_stack.peek())
+
+@is_circuit_code
```
Comment (scroll-dev) [inline src/zkevm_specs/opcode/stack.py:63]: move the test to `zkevm-specs/tests/test_opcode.py`

File: src/zkevm_specs/opcode/stack.py [hunk 2]
```diff
@@ -48,4 +58,19 @@ def check_dupx():
...
+    word2:U256 = 30
+    word3:U256 = 40
+    evm_stack.push(word1)
+    evm_stack.push(word2)
+    evm_stack.push(word3)
+    evm_stack.swapx(2)
+    assert(word1 == evm_stack.peek())
+    evm_stack.pop()
+    evm_stack.pop()
+    assert(word3 == evm_stack.peek())
```
Comment (scroll-dev) [inline src/zkevm_specs/opcode/stack.py:76]: add EOL

File: src/zkevm_specs/opcode/stack.py (hunk 1 above)
Comment (DreamWuGit) [reply src/zkevm_specs/opcode/stack.py]: udpated

**Comment by DreamWuGit** [comment]
closing as merged from upstream

**Pull request closed without merging**